select.js

```diff
@@ -0,0 +1,14 @@
+define(function() {
+    var init = function() {
+        $('body').on('change', '.c-select', function() {
```

Comment:
UI script is not ready for review ATM, so let’s hold off that. However, for Stencil I think it’s totally safe to have js rely on classnames. The reason for `js-` classes is if you have behaviour x relying on class y, where x and y are implemented separately. This is the case for, e.g. some JS module that might be integrated with any old HTML. However, a component is a single unit, so we shouldn’t need `js-` classes.